src/apps/bridge/bridgePowerController.cpp

```diff
@@ -185,46 +201,59 @@ void BridgePowerController::_update(
       vPortFree(printbuf);
       BRIDGE_LOG_PRINT("Bridge State Init Complete\n");
       checkAndUpdateRTC();
+      uint32_t currentCycleS = getEpochS();
+      if (_rtcSet && _sampleIntervalStartS > currentCycleS) {
+        time_to_sleep_ms =
+            MAX((_sampleIntervalStartS - currentCycleS) * 1000, MIN_TASK_SLEEP_MS);
+        // The default state until first sample interval depends
+        // on whether bus power control is enabled.
+        powerBusAndSetSignal(!_powerControlEnabled);
+      }
```

Comment:
This handles aligning the first sample to UTC in the case where the RTC has already been set by the time the init period ends.